### DISCUSSION TITLE
dockge can not manage mailcow stack

### DISCUSSION THREAD

**Discussion by meehr** [louislam/dockge#382] [Ask for Help]
### ⚠️ Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/dockge/security/policy)

### 📝 Describe your problem

I have installed Dockge on a new server with a Traefik - Crowdsec stack. 

These two stacks are also recognized by Dockge and can be managed. 

I then downloaded Mailcow and the stack is also recognized. However, dockge says that it cannot manage it.

In the log of dockge I also have the following error message. 

Does anyone know a solution?

### 📝 Error Message(s) or Log

dockge-dockge-1  | 2024-01-20T00:43:13Z [GETSERVICESTATUSLIST] ERROR: Error: spawn docker ENOENT
dockge-dockge-1  |     at __node_internal_captureLargerStackTrace (node:internal/errors:496:5)
dockge-dockge-1  |     at __node_internal_errnoException (node:internal/errors:623:12)
dockge-dockge-1  |     at ChildProcess._handle.onexit (node:internal/child_process:283:19)
dockge-dockge-1  |     at onErrorNT (node:internal/child_process:476:16)
dockge-dockge-1  |     at process.processTicksAndRejections (node:internal/process/task_queues:82:21) {
dockge-dockge-1  |   errno: -2,
dockge-dockge-1  |   code: 'ENOENT',
dockge-dockge-1  |   syscall: 'spawn docker',
dockge-dockge-1  |   path: 'docker',
dockge-dockge-1  |   spawnargs: [ 'compose', 'ps', '--format', 'json' ]
dockge-dockge-1  | }

### 🐻 Dockge Version

latest

### 💻 Operating System and Arch

Debian 12 arm

### 🌐 Browser

Chrome

### 🐋 Docker Version

25.0.0

**Comment by lohrbini** [comment]
Since moby has issues in v25.0.0 with start and stopping containers try upgrade to a later version 

https://github.com/moby/moby/pull/47171
